Source/WebKit/Shared/WebCoreArgumentCoders.serialization.in

```diff
@@ -1768,12 +1768,26 @@ header: <WebCore/ScrollingCoordinatorTypes.h>
     bool useDarkAppearanceForScrollbars;
 };
 
+[CustomHeader] struct WebCore::RequestedKeyboardScrollData {
```

Comment:
Nit: Let's add `header: <WebCore/KeyboardScroll.h>` above this line for consistency.

Reply:
@pxlcoder but `RequestedKeyboardScrollData` isn't in `KeyboardScroll.h` ?

Reply:
sorry, `header: <WebCore/ScrollingCoordinatorTypes.h>`